app/Http/Controllers/CourseController.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Interns2024c\Http\Controllers;
+
+use Illuminate\Foundation\Auth\Access\AuthorizesRequests;
+use Illuminate\Http\Request;
+use Inertia\Inertia;
+use Interns2024c\Models\Course;
+
+class CourseController extends Controller
+{
+    use AuthorizesRequests;
+
+    public function index(Request $request)
+    {
+
+        $query = Course::query();
+
+
```

Comment:
Please remove unnecessary lines - if you run `make fix` it should be done automatically